### DISCUSSION TITLE
Save APIKEY_PATH to PostgreSQL DB?

### DISCUSSION THREAD

**Discussion by dkindlund** [FlowiseAI/Flowise#3211] [Q&A] [answered]
Hey @HenryHengZJ, are there any plans to store APIKEY data to a PostgreSQL DB instead of to a file on disk?  The reason I ask, is that I can't run Flowise in an AWS App Runner container, because there's no way to attach persistent volumes to the container -- so every time the container gets upgraded/re-deployed, the local files storing APIKEY material gets wiped and reset.

**Comment by HenryHengZJ** [comment, marked as answer]
here you go: https://docs.flowiseai.com/configuration/environment-variables#for-flowise-api-keys

**Reply by dkindlund** [reply]
That's awesome, @HenryHengZJ -- thank you!  Which version of Flowise supports this new feature, btw?
